sqlitecluster/SQLite.cpp

```diff
@@ -298,11 +298,20 @@ string SQLite::read(const string& query) {
 }
 
 bool SQLite::read(const string& query, SQResult& result) {
+    
     // Execute the read-only query
-    SASSERTWARN(!SContains(SToUpper(query), "INSERT "));
-    SASSERTWARN(!SContains(SToUpper(query), "UPDATE "));
-    SASSERTWARN(!SContains(SToUpper(query), "DELETE "));
-    SASSERTWARN(!SContains(SToUpper(query), "REPLACE "));
+    if (!SContains(SToUpper(query), "UPDATE ")) {
```

Comment:
Don't you want to log something if `SContains(SToUpper(query), "UPDATE ")` ? 😕 
I'm kind of even questioning why we'd want that. If you still want it, please use something other than exactly what `SASSERTWARN` would output. (eg `SHMMM("Query contains 'UPDATE'")`). There's not assertion that failed if you removed the assertion.

Reply:
good point